central/globaldb/postgres.go

```diff
@@ -104,6 +97,37 @@ func GetPostgres() *pgxpool.Pool {
 	return postgresDB
 }
 
+// GetPostgresConfig - gets the configuration used to connect to Postgres
+func GetPostgresConfig() (map[string]string, *pgxpool.Config, error) {
+	centralConfig := config.GetConfig()
+	password, err := os.ReadFile(dbPasswordFile)
+	if err != nil {
+		log.Fatalf("pgsql: could not load password file %q: %v", dbPasswordFile, err)
```

Comment:
I know it was extracted but this function returns error so instead of fatal we should return an error and exit in the caller